en/services/gimbal_v2.md

```diff
@@ -0,0 +1,374 @@
+
+A particular MAVLink component can implement multiple gimbal managers (e.g. an autopilot can implement two gimbal managers in order to control two gimbal devices).
+
+
+#### Addressing of Gimbal Devices {#gimbal_device_addressing}
+
+*Gimbal Manager* commands and messages have a param field to indicate the component ID of the *Gimbal Device* that they intend to control.
+
+A system that wants to control a *particular* gimbal device will send messages to the component that has the manager(s), specifying the particular device to be controlled.
+
+If all gimbal devices should be controlled (on the component that has the gimbal managers), this param/field can be set to 0 (signalling "all").
+
+
+## Implementation and Messages
+
+### Messages between Ground Station and Gimbal Manager
+
+#### Discovery of Gimbal Manager
+
+A ground station should initially discover all gimbal managers by sending a broadcast [MAV_CMD_REQUEST_MESSAGE](#MAV_CMD_REQUEST_MESSAGE) for [GIMBAL_MANAGER_INFORMATION](#GIMBAL_MANAGER_INFORMATION).
+Every gimbal manager should respond with `GIMBAL_MANAGER_INFORMATION`.
+
+The `GIMBAL_MANAGER_INFORMATION` contains important information such as gimbal capabilities ([GIMBAL_MANAGER_CAP_FLAGS](#GIMBAL_MANAGER_CAP_FLAGS)), maximum angles and angle rates, as well as the `gimbal_component` which is the component ID of the *Gimbal Device* controlled by this *Gimbal Manager*.
+
+#### Gimbal Manager Status
+
+A *Gimbal Manager* should send out [GIMBAL_MANAGER_STATUS](#GIMBAL_MANAGER_STATUS) at a low regular rate (e.g. 5 Hz) to inform the ground station about its status.
+
+#### Manually Controlling a Gimbal Using MAVLink
+
+A ground station can manually control a gimbal by sending [GIMBAL_MANAGER_SET_ATTITUDE](#GIMBAL_MANAGER_SET_ATTITUDE).
+This allows controlling the gimbal with angles, or angular rates, or both.
+
+
+### Messages between Gimbal Manager and Gimbal Device
+
+#### Discovery of Gimbal Device
+
+The MAVlink node where the *Gimbal Manager* is implemented needs to discover *Gimbal Devices* by sending a broadcast [MAV_CMD_REQUEST_MESSAGE](#MAV_CMD_REQUEST_MESSAGE_2) for [GIMBAL_DEVICE_INFORMATION](#GIMBAL_DEVICE_INFORMATION).
+Every gimbal device should respond with `GIMBAL_DEVICE_INFORMATION`.
+
+The MAVLink node should then create as many *Gimbal Manager* instances as *Gimbal Devices* found.
+
+#### Control of a Gimbal Device
+
+To control the angle and/or angular rate of the *Gimbal Device*, use the message [GIMBAL_DEVICE_SET_ATTITUDE](#GIMBAL_DEVICE_SET_ATTITUDE).
+If the gimbal manager has multiple gimbal control inputs available it should deconflict them as explained below.
+
+#### Autopilot State for Gimbal Device
+
+The autopilot should also send the message [AUTOPILOT_STATE_FOR_GIMBAL_DEVICE](#AUTOPILOT_STATE_FOR_GIMBAL_DEVICE) to the gimbal device.
```

Comment:
When should this be sent and at what rate?

E.g. I know this is needed for tracking, so presumably at high rate. So presumably you sent it only on tracking modes? Also not clear what happens if you set ROI (and need to track) and the device doesn't support this.

Reply:
This has nothing to do with tracking as such. It's just information that the gimbal needs.

Reply:
@julianoes Can you update the docs with indication of the rate, and confirmation that it should be sent all the time (assuming that is the case)? 

FWIW that doesn't "feel right" to me - i.e. I assume not every gimbal can do anything with this info. But it doesn't matter - we just need to say.